server/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java

```diff
@@ -589,6 +590,14 @@ public void testAlterTableAddObjectColumnToExistingObject() {
             fqColumnNames.add((String) row[0]);
         }
         assertThat(fqColumnNames).containsExactly("o", "o['x']", "o['y']");
+
+        String expectedMapping = "{\"default\":" +
+            "{\"dynamic\":\"strict\",\"_meta\":{}," +
+            "\"properties\":{\"o\":{\"position\":1,\"oid\":1,\"dynamic\":\"true\"," +
+            "\"properties\":" +
+            "{\"x\":{\"type\":\"keyword\",\"position\":2,\"oid\":2}," +
+            "\"y\":{\"type\":\"integer\",\"position\":3,\"oid\":3}}}}}}";
```

Comment:
Max OID used to be 2 (due to non-propagating updated OID). Added mapping assertion to the object case.

This is not related to `test_concurrent_statements_that_add_columns_to_partitioned_table_result_in_dynamic_mapping_updates` failure which I'm still looking into